### PR TITLE
i18n: 🇵🇱

### PR DESCRIPTION
**Pull request by m4sk1n**

### CODE REVIEW

File: config/locales/pl.yml [hunk 1]
```diff
@@ -459,7 +459,7 @@ pl:
       expires_at: Wygaśnie po
       uses: Użycia
     title: Zaproś użytkowników
-  keyword_mutes:
+  keyword_mutes: Wyciszone słowa kluczowe
```
Comment (yipdw) [inline config/locales/pl.yml]: The translated string should be removed from here; this key starts a YAML map.

File: config/locales/pl.yml [hunk 2]
```diff
@@ -692,6 +694,8 @@ pl:
 
       <p>Tekst bazuje na <a href="https://github.com/discourse/discourse">polityce prywatności Discourse</a>.</p>
     title: Zasady korzystania i polityka prywatności %{instance}
+  themes:
+    default: Mastodon
```
Comment (marrus-sh) [inline config/locales/pl.yml]: We actually don't need the themes map anymore; we don't use it and Travis will fuss at us if it's included I think.

**Comment by beatrix-bitrot** [comment]
@m4sk1n any interest in this one? it has some issues mentioned in comments, as well as conflicts now

**Comment by m4sk1n** [comment]
@beatrix-bitrot I'll do it today

**Comment by m4sk1n** [comment]
@beatrix-bitrot, I’m waiting for merge

**Pull request merged**